aten/src/ATen/native/Convolution.cpp

```diff
@@ -1223,12 +1223,9 @@ static ConvBackend _select_conv_backend(
       return ConvBackend::Cudnn;
     } else if (params.use_miopen(input, weight, bias_sizes_opt.has_value())) {
       return ConvBackend::MiopenDepthwise;
-    } else if (params.use_mkldnn(input, weight)) {
```

Comment:
Remove this `if` branch as it is wrongly introduced in https://github.com/pytorch/pytorch/pull/149114.

Reply:
Thanks for helping fix my mistake.